### PR TITLE
Update to 4.0.70

### PR DESCRIPTION
**Pull request by duramato**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
+### 4.0.70 (2015-10-02)
+
+[full changelog](https://github.com/SiCKRAGETV/SickRage/compare/v4.0.69.1...v4.0.70)
+
+* Added: Different muxed release types to match qualities
```
Comment (MGaetan89) [inline CHANGES.md]: I guess you meant `mixed`?

Comment (duramato) [inline CHANGES.md]: Nope ,muxed are releases with embebed subs much like dubbed is for audio

Comment (MGaetan89) [inline CHANGES.md]: Ok, didn't know that. Thanks for the detail ;)

Comment (duramato) [inline CHANGES.md]: @MGaetan89 just to add up :p 
It's like VOSTFR ones or KORSUB

**Pull request merged**